provisioner/template.go

```diff
@@ -655,6 +664,7 @@ func poolsDistributed(dedicated string, pools []*api.NodePool) bool {
 //   - are correctly configured with regards to the labels and taints
 //   - don't have AZ restrictions
 //   - use the worker-splitaz profile.
+//   - use the worker-karpenter profile.
```

Comment:
Based on the implementation should this rather be "- use the worker-splitaz or worker-karpenter profile."?